.github/workflows/ci.yml

```diff
@@ -0,0 +1,142 @@
+name: CI testing
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+      - release-candidate-*
+    tags:
+      - v*
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        description: Run the build with SSH debugging enabled
+        required: false
+        default: false
+
+env:
+  NUMPY: numpy
```

Comment:
Is this needed?

Reply:
At one point I had to add it to one of the tests to fix an issue, but I might have done that in places where it didn't need it too, I'll check.